editor/src/components/inspector/common/css-utils.ts

```diff
@@ -4500,6 +4503,7 @@ const cssPrinters: CSSPrinters = {
   height: printCSSNumberOrUndefinedAsAttributeValue('px'),
   flexBasis: printCSSNumberOrUndefinedAsAttributeValue('px'),
   gap: printCSSNumberAsAttributeValue('px'),
+  zIndex: printCSSNumberOrUndefinedAsAttributeValue('px'), // TODO
```

Comment:
This should definitely not have a unit!

Reply:
Thanks! fixed in [82a032a](https://github.com/concrete-utopia/utopia/pull/3668/commits/82a032a9dabe0b48fcaa35f17a2c65400d28ad5d)